packages/host/app/components/operator-mode/container.gts

```diff
@@ -590,9 +607,60 @@ export default class OperatorModeContainer extends Component<Signature> {
   };
 
   @action updateSubmode(submode: Submode) {
+    switch (submode) {
+      case Submode.Interact:
+        this.onCodePathChange(null);
+        break;
+      case Submode.Code:
+        this.onCodePathChange(new URL(this.lastCardInRightMostStack.id));
+        break;
+    }
+
```

Comment:
please include a default case. it should assert not never. this is a good type script practice that will catch issues if we ever add a new submode. an example is here:

https://github.com/cardstack/boxel/blob/549a5b72cc8aa978e4c5ab15fdb602b9a6b54391/packages/runtime-common/loader.ts#L722-L723